doc/python/format.rst

```diff
@@ -0,0 +1,80 @@
+.. _pycablecellformat:
+
+Description Format
+==================
+
+Arbor provides readers and writers for describing :ref:`label dictionaries <labels>`,
+:ref:`decoration objects <cablecell-decoration>`, :ref:`morphologies <morph>` and
+:ref:`cable cells <cablecell>`, referred to here as *arbor-components*.
+
+A detailed description of the s-expression format used to describe each of these components
+can be found :ref:`here <formatcablecell>`.
+
+The arbor-components and meta-data
+----------------------------------
+.. currentmodule:: arbor
+
+.. py:class:: meta_data
+
+   .. py:attribute:: int version
+
+      Stores the version of the format being used.
+
+.. py:class:: cable_cell_component
+
+   .. py:attribute:: meta_data meta
+
+      Stores meta-data pertaining to the description of a cable cell component.
+
+   .. py:attribute:: component
+
+      Stores one of :class:`decor`, :class:`label_dict`, :class:`morphology` or :class:`cable_cell`.
+
+Reading arbor-components
```

Comment:
I think that you combine reading and writing here, as opposed to the C++ section. if so, the heading should reflect that.